docs/generate-evidence-strings.md

```diff
@@ -169,7 +175,7 @@ If everything has been done correctly, hash sums will be the same. Note that the
   + Generated evidence strings validate against the schema
     - Have actually been submitted to the Open Targets cloud storage
     - The file is the same as on the cluster (check md5)
-    - E-mail has been sent to Open Targets, with eva-dev in copy
+    - E-mail has been sent to Open Targets, with opentargets-clinvar in copy
```

Comment:
Same as before, if this email address can be disclosed in this document, I would add it.

Reply:
I don't know if this one is public yet, it's probably fine (and actually probably should be added to the main readme as the primary contact for this repo) but just in case I left it off for now.